collector/diskstats_linux.go

```diff
@@ -206,6 +225,70 @@ func getDiskStats() (map[string]map[int]string, error) {
 	return parseDiskStats(file)
 }
 
+func getDiskSectorSize(dev string) (string, error) {
+	file, err := os.Open(sysFilePath(fmt.Sprintf("class/block/%s/queue/hw_sector_size", dev)))
+	if err != nil {
+		return "", err
+	}
+	defer file.Close()
+
+	return parseDiskSectorSize(file)
+}
+
+func parseDiskSectorSize(r io.Reader) (string, error) {
+	size := make([]byte, 16)
+	n, err := r.Read(size)
+	if err != nil {
+		return "", err
+	}
+
+	return string(size[0 : n-1]), nil // strip newline
+}
+
+func convertDiskSectorsToBytes(sectorCount string, sectorSize string) (string, error) {
+	sectors, err := strconv.ParseInt(sectorCount, 10, 64)
+	if err != nil {
+		return "", err
+	}
+	size, err := strconv.ParseInt(sectorSize, 10, 64)
+	if err != nil {
+		return "", err
+	}
+
+	return strconv.FormatInt(sectors*size, 10), nil
```

Comment:
We're already hardcoding it in the default console templates. It seems to me something that's very unlikely to change, and when it does we can worry about it then.